### PR TITLE
Minor tweaks to KNX docs

### PR DESCRIPTION
**Pull request by spacegaier**
## Type of change

- [X] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by farmio — DISMISSED** [review]

File: source/_integrations/knx.markdown [hunk 1]
```diff
@@ -442,29 +442,29 @@ knx:
...
 
 The following values are valid for the `hvac_mode` attribute:
 
-- Off (maps internally to HVAC_MODE_OFF within Home Assistant)
-- Auto (maps internally to HVAC_MODE_AUTO within Home Assistant)
-- Heat (maps internally to HVAC_MDOE_HEAT within Home Assistant)
-- Cool (maps internally to HVAC_MDOE_COOL within Home Assistant)
-- Fan only (maps internally to HVAC_MODE_FAN_ONLY within Home Assistant)
-- Dry (maps internally to HVAC_MODE_DRY within Home Assistant)
+- `Off `(maps internally to `HVAC_MODE_OFF` within Home Assistant)
```
Comment (marvin-w) [inline source/_integrations/knx.markdown:454]:
```suggestion
- `Off`(maps internally to `HVAC_MODE_OFF` within Home Assistant)
```

Comment (marvin-w) [inline source/_integrations/knx.markdown]:
```suggestion
- `Off` (maps internally to `HVAC_MODE_OFF` within Home Assistant)
```

**Review by marvin-w — CHANGES_REQUESTED** [review]
Looks good, just some minor typo! Thanks!

**Review by frenck — APPROVED** [review]
Thanks, @spacegaier 👍

**Pull request merged**